common/main/de_LI.xml

```diff
@@ -46,6 +46,9 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 					<pattern>¤ #,##0.00</pattern>
 					<pattern alt="noCurrency" draft="provisional">↑↑↑</pattern>
 				</currencyFormat>
+				<currencyFormat type="accounting">
+					<pattern>¤ #,##0.00</pattern>
+				</currencyFormat>
```

Comment:
Here is another example where there was previously no value.

accounting-currency inherits laterally from standard-currency; "¤ #,##0.00" matches bailey

https://st.unicode.org/cldr-apps/v#/de_LI/Number_Formatting_Patterns/79c89d7e54153863

https://st.unicode.org/cldr-apps/v#/de_LI/Number_Formatting_Patterns/53687a25c19b6481